editor/osm_auth.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "std/string.hpp"
+#include "std/utility.hpp"
+
+namespace osm
+{
+
+using TKeySecret = pair<string /*key*/, string /*secret*/>;
```

Comment:
We initially tried this approach and later changed it to std::pair, to avoid unnecessary dependency for external code in some cases.